### PR TITLE
Add mappings for v1 types to v2 categories

### LINKED ISSUE #300: Migrate v1 types to v2 categories
Existing v1 to v2 mappers do not support the type/category filtering used by the web service (see https://github.com/openmsupply/unified-codes/pull/298#discussion_r527270095).

## Expected Behaviour

Web service should fetch +filter data correctly.

## Current Behaviour

Web service is not fetching or filtering v2 data.

## Possible Solution

- Add mapper for mapping v1 types to v2 categories.
- Update entity query to filter by parent category.

### PR DESCRIPTION
**Pull request by wlthomson**
Fixes #300.

## Description

Maps v1 types to v2 categories for compatibility with web frontend.

### Documentation:

Tick below if one of the following applies:

1. Documentation has been added to cover changes introduced by this PR.
2. This PR requires no changes to documentation.

- [ ] Documentation OK.

### Tests:

Tick below if one of the following applies:

1. Tests have been added to cover changes introduced by this PR.
2. This PR requires no new tests.

- [ ] Tests OK.

### CODE REVIEW

File: unified-codes/apps/data-service/src/app/v1/types/DgraphDataSource.ts [hunk 1]
```diff
@@ -64,7 +70,6 @@ export class DgraphDataSource extends RESTDataSource {
       query (func: eq(type, "drug")) @cascade {
         code
         description
-        type
```
Comment (wlthomson) [inline unified-codes/apps/data-service/src/app/v1/types/DgraphDataSource.ts:67]: Wha? I don't remember doing this. Guess can fix it in the merge...

File: unified-codes/apps/data-service/src/app/v1/types/DgraphDataSource.ts [hunk 2]
```diff
@@ -36,14 +36,20 @@ export class DgraphDataSource extends RESTDataSource {
...
+          return 'Drug';
+        case EEntityType.MEDICINAL_PRODUCT:
+          return 'Consumable';
+        case EEntityType.OTHER:
+          return 'Other';
+        default:
+          return type;
+      }
+    });
+    return JSON.stringify(categories);
```
Comment (wlthomson) [inline unified-codes/apps/data-service/src/app/v1/types/DgraphDataSource.ts:52]: Idea is to return types as a stringified array, makes the query a bit tidier!

Comment (katherine-sussol) [reply unified-codes/apps/data-service/src/app/v1/types/DgraphDataSource.ts:52]: Nice, works well 👍

File: unified-codes/apps/data-service/src/app/v1/types/DgraphDataSource.ts (hunk 1 above)
Comment (katherine-sussol) [reply unified-codes/apps/data-service/src/app/v1/types/DgraphDataSource.ts:67]: 🙀

**Review by katherine-sussol — APPROVED** [review]
Looks good to me, (merged in merge in master to test things - once that was done everything was working nicely 👍 ).

**Pull request merged**